application/account-management/Api/Api.csproj

```diff
@@ -31,4 +31,8 @@
         </Content>
     </ItemGroup>
 
+    <Target Name="CreateSwaggerJson" AfterTargets="Build" Condition="$(Configuration)=='Debug'">
```

Comment:
might remove or update the condition?

Reply:
I think we should build it also on the server. And either leave out the `schema.d.ts` from Git, or fail the build if this file is changed.

Reply:
I've excluded the generated schema file from git